src/System.Runtime/tests/System/Runtime/Serialization/OnSerializingAttributeTests.cs

```diff
@@ -0,0 +1,17 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Xunit;
+
+namespace System.Runtime.Serialization.Tests
+{
+    public class OnSerializingAttributeTests
+    {
+        [Fact]
+        public void Ctor_Default()
+        {
+            new OnSerializingAttribute();
```

Comment:
Do u plan to add more tests to these classes ? if not, combine all the ctor tests in one file